testprojects/pants-plugins/src/python/test_pants_plugin/register.py

```diff
@@ -7,7 +7,9 @@
 
```

Comment:
Per previous discussions about howitzers and integration tests, what do you think about reverting everything from here down and just adding a unit test? The needed infra is here afaict to do this easily and at great runtime speed:
https://github.com/pantsbuild/pants/blob/d14943d999fa15a13f0ec9d29821bf1212294ffc/tests/python/pants_test/init/test_extension_loader.py#L255

Reply:
Will do! Was hacking on several things today and didn't spend quite enough time making this mergeable.

Reply:
I have moved the testing into `test_extension_loader.py`, which is absolutely all that is necessary for this case, as many permutations of rules are used in the rest of the codebase. Travis will hopefully be happier now.